set.mm

```diff
@@ -209495,6 +209496,60 @@ is always a subcategory (and it is full, meaning that all morphisms of
       QWJBFVRWKWLWNWOVCZVDVQXBXCXDUTVEVQVSBVFPNWAVNULXFEBVSVGUJVHAVPVIVJVKVL $.
   $}
 
+  ${
+    $d B w x y z $.  $d H w x y z $.  $d Y w $.
+    cat1lem.1 $e |- C = ( SetCat ` U ) $.
+    cat1lem.2 $e |- ( ph -> U e. V ) $.
+    cat1lem.3 $e |- B = ( Base ` C ) $.
+    cat1lem.4 $e |- H = ( Hom ` C ) $.
+    cat1lem.5 $e |- ( ph -> (/) e. U ) $.
+    cat1lem.6 $e |- ( ph -> Y e. U ) $.
+    cat1lem.7 $e |- ( ph -> (/) =/= Y ) $.
+    $( The category of sets in a universe containing the empty set and another
```

Comment:
```suggestion
    $( The category of sets in a "universe" containing the empty set and another
```
Quotes, since this is not a universe in any standard sense.